### PR TITLE
AT_01.02.02 | Start page > Login button: visible / clickable / opening Login popup

### PR DESCRIPTION
**Pull request by Bannikova77**
https://trello.com/c/gy4pA8tN/153-at010202-start-page-login-button-visible-clickable-opening-login-popup

### CODE REVIEW

File: cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.02_Login_register_section_elements.cy.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/// <reference types="Cypress" />
+
+import {StartPage} from "../../../../pageObjects/StartPage.js";
+import {LoginPopup} from "../../../../pageObjects/StartPage.js";
+
+const startPage = new StartPage();
+const loginPopup = new LoginPopup();
+
+
+describe('US_01.02 |  Login register section elements', () => {
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.02_Login_register_section_elements.cy.js:10]: Leave 1 space after "|"

File: cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.02_Login_register_section_elements.cy.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    beforeEach(function () {
+        cy.fixture('startPage/headers').then(headers => {
+            this.headers = headers;
+        });
+
+        cy.visit('/')
+    });
+
+    it('AT_01.02.02 | Start page > Login button: visible / clickable / opening Login popup', function () {
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.02_Login_register_section_elements.cy.js:20]: The name of the test should only contain the AT card number and the name of the test.
it('AT_01.02.02 | Login button: visible / clickable / opening Login popup'

File: cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.02_Login_register_section_elements.cy.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/// <reference types="Cypress" />
+
+import {StartPage} from "../../../../pageObjects/StartPage.js";
+import {LoginPopup} from "../../../../pageObjects/StartPage.js";
+
+const startPage = new StartPage();
+const loginPopup = new LoginPopup();
+
+
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.02_Login_register_section_elements.cy.js:9]: You can remove 1 empty line

File: cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.02_Login_register_section_elements.cy.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+describe('US_01.02 |  Login register section elements', () => {
+
+    beforeEach(function () {
+        cy.fixture('startPage/headers').then(headers => {
+            this.headers = headers;
+        });
+
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.02_Login_register_section_elements.cy.js:16]: You can remove empty line

File: cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.02_Login_register_section_elements.cy.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    it('AT_01.02.02 | Start page > Login button: visible / clickable / opening Login popup', function () {
+
+        startPage.getLoginButton().should('be.visible');
+        startPage.clickLoginButton();
+        loginPopup.getHeaderText().should('include.text', this.headers.header_Login_Popup.text)
+    });
+})
+
+
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.02_Login_register_section_elements.cy.js:28]: You can remove 1 empty line

**Review by Rom-T — APPROVED** [review]
Good Job!

**Comment by elviraKh1** [comment]
@Bannikova77 Svetlana, your file is in the incorrect folder. You should refactor your code .
image

**Pull request merged**